### PR TITLE
Added `skip` docs

### PR DESCRIPTION
**Pull request by Radinyn**

### CODE REVIEW

File: website/docs/tutorials/07-testing/02-cheatcodes/skip.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# `skip`
+```python
+def skip(reason: Optional[str] = None) -> None:
+```
+Skip a test.
+
+:::warning
+This cheatcode is only available in test's setup.
```
Comment (MaksymilianDemitraszek) [inline website/docs/tutorials/07-testing/02-cheatcodes/skip.md:8]:
```suggestion
This cheatcode is only available in test's setup case.
```
please link relevant section of docs

File: website/docs/tutorials/07-testing/02-cheatcodes/skip.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+func test_function() {
+    // %{ skip("Reason") %} <-- won't work
+    return ();
+}
+
+@external
+func __setup__() {
+    // %{ skip("Reason") %} <-- won't work
+    return ();
+}
```
Comment (MaksymilianDemitraszek) [inline website/docs/tutorials/07-testing/02-cheatcodes/skip.md:30]:
```suggestion
@external
func __setup__() {
    // %{ skip("Reason") %} <-- won't work
    return ();
}

@external
func setup_function() {
    %{ skip("Reason") %} // <-- ok
    return ();
}

@external
func test_function() {
    // %{ skip("Reason") %} <-- won't work
    return ();
}
```

**Review by MaksymilianDemitraszek — CHANGES_REQUESTED** [review]

**Comment by mkaput** [comment]
Please revert changes to yarn lock

File: website/docs/tutorials/07-testing/02-cheatcodes/skip.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+```cairo
+%lang starknet
+
+@external
+func setup_function() {
+    %{ skip("Reason") %} // <-- ok
+    return ();
+}
+
+@external
```
Comment (mkaput) [inline website/docs/tutorials/07-testing/02-cheatcodes/skip.md:20]: You have already said that in the warning block. No need to repeat info in example :)

File: website/docs/tutorials/07-testing/02-cheatcodes/skip.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# `skip`
+```python
+def skip(reason: Optional[str] = None) -> None:
+```
+Skip a test.
```
Comment (mkaput) [inline website/docs/tutorials/07-testing/02-cheatcodes/skip.md:5]: I would like to have info here why is this useful.

**Review by mkaput — CHANGES_REQUESTED** [review]

File: website/docs/tutorials/07-testing/02-cheatcodes/skip.md (hunk 3 above)
Comment (Radinyn) [reply website/docs/tutorials/07-testing/02-cheatcodes/skip.md:20]: To be honest I'm not sure, it's easier to find it here.

File: website/docs/tutorials/07-testing/02-cheatcodes/skip.md (hunk 4 above)
Comment (Radinyn) [reply website/docs/tutorials/07-testing/02-cheatcodes/skip.md]: Done

File: website/docs/tutorials/07-testing/02-cheatcodes/skip.md (hunk 3 above)
Comment (mkaput) [reply website/docs/tutorials/07-testing/02-cheatcodes/skip.md:20]: I don't share this opinion. Would love to hear from @MaksymilianDemitraszek what he thinks about this.

File: website/docs/tutorials/07-testing/02-cheatcodes/skip.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    %{ skip("Reason") %} // <-- ok
+    return ();
+}
+
+@external
+func test_function() {
+    // %{ skip("Reason") %} <-- won't work
+    return ();
+}
+```
```
Comment (mkaput) [inline website/docs/tutorials/07-testing/02-cheatcodes/skip.md:31]: Hmm I think it would be also nice to provide a described example how to do conditional skipping.

**Review by mkaput — CHANGES_REQUESTED** [review]

File: website/docs/tutorials/07-testing/02-cheatcodes/skip.md (hunk 3 above)
Comment (MaksymilianDemitraszek) [reply website/docs/tutorials/07-testing/02-cheatcodes/skip.md:24]: I agree with @mkaput , it is a bit wordy to include 2/3 of example code just to show it doesn't work like that

Comment (MaksymilianDemitraszek) [reply website/docs/tutorials/07-testing/02-cheatcodes/skip.md:24]: It is nice you tried explain it carefully, but it is generally nice when we keep [explaining in details measure] even in whole docs, and we don't provide such examples for other cheatcodes

Comment (Radinyn) [reply website/docs/tutorials/07-testing/02-cheatcodes/skip.md:24]: Done

File: website/docs/tutorials/07-testing/02-cheatcodes/skip.md (hunk 5 above)
Comment (Radinyn) [reply website/docs/tutorials/07-testing/02-cheatcodes/skip.md:28]: Done

File: website/yarn.lock [hunk 6]
```diff
@@ -4,45 +4,45 @@
 
 "@algolia/autocomplete-core@1.5.2":
   version "1.5.2"
-  resolved "https://registry.yarnpkg.com/@algolia/autocomplete-core/-/autocomplete-core-1.5.2.tgz#ec0178e07b44fd74a057728ac157291b26cecf37"
```
Comment (mkaput) [inline website/yarn.lock:7]: could you please move changes to this file to separate PR which updates deps?

**Review by mkaput — CHANGES_REQUESTED** [review]

File: website/yarn.lock (hunk 6 above)
Comment (Radinyn) [reply website/yarn.lock:7]: Surprisingly when I just simply updated the docs from the master branch there seem to be no changes.

Comment (mkaput) [reply website/yarn.lock:7]: So plz remove this file from here :)

Comment (Radinyn) [reply website/yarn.lock]: Done

**Review by mkaput — APPROVED** [review]

**Comment by mkaput** [comment]
LGTM
